Tests/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ macro(setup_test)
     set(PLOT_TEST ${CURRENT_TEST_BINARY_DIR}/plt00010)
     # Find fcompare
     if(PELEC_ENABLE_FCOMPARE_FOR_TESTS)
-      set(FCOMPARE ${CMAKE_BINARY_DIR}/Submodules/AMReX/Tools/Plotfile/fcompare)
+      set(FCOMPARE ${CMAKE_BINARY_DIR}/Submodules/PelePhysics/amrex/Tools/Plotfile/fcompare)
```

Comment:
Shouldn't this be `Submodules/PelePhysics/Submodules/amrex`?

Reply:
yes! good catch. It explains some of the failures I was seeing on the CI. Thank you.